### ISSUE TITLE
Creating two request instead one

### ISSUE THREAD

**Issue by mrZherart** [google/google-api-php-client-services#103]
Hi, I think you have a bug in library, because code creating two same folders instead one (twice putting data in google spreadsheets). But I call creating folder / putting data function only once.

On this example showing creating folder.
I created server-to-server auth and its work fine (I add service account to my Team Drive and use service_account.json credentials for connect). I using code on my local server for testing code with local domain.

On pict under you can see my class with connection in constructor and method creating of directory.

![93f09a6957](https://user-images.githubusercontent.com/4329958/35653149-6fe89014-06ef-11e8-8898-1f255bf522db.jpg)

The client code looks like this:

```
require_once 'teamdrive.class.php';

$td = new TeamDrive;
$td->create_project_folder('TEST_PROJECT');
```

On pict drive.jpeg you can see the result of once(!) call the code below. We have created two folders, and am really don`t now why...

![drive](https://user-images.githubusercontent.com/4329958/35653076-0fba8d8c-06ef-11e8-95c7-e818d15f7c6b.jpeg)

Also you can see tech service member on screenhot under this text.

![members](https://user-images.githubusercontent.com/4329958/35653097-272f5524-06ef-11e8-908d-18d451e8acf3.jpg)

Thanks for your help.

**Comment by mattwhisenhunt** [comment]
Duplicate of #44

**Comment by mrZherart** [comment]
I noticed that this is happening only with names with underscore like "name_folder". With one word names all okey. Very strange.

**Comment by mrZherart** [comment]
The bug was in my server config. The nginx server executed the file twice. Sorry for the misinformation.
